src/ni/vsbuild/Pipeline.groovy

```diff
@@ -207,9 +207,14 @@ class Pipeline implements Serializable {
          return
       }
 
+      def archiveParentLocation = Archive.calculateArchiveParentLocation(script, configuration)
       def archiveLocation = Archive.calculateArchiveLocation(script, configuration)
       script.node(pipelineInformation.nodeLabel) {
          script.stage('validateShouldBuildPipeline') {
+            if (script.fileExists(archiveParentLocation + "\\norebuild")) {
```

Comment:
I considered just walking up the directory tree from `archiveLocation` to get to this folder (e.g. `archiveLocation + "\\..\\norebuild"`, but this felt more explicit.